llvm/lib/IR/Core.cpp

```diff
@@ -3769,6 +3771,8 @@ void LLVMSetOrdering(LLVMValueRef MemAccessInst, LLVMAtomicOrdering Ordering) {
 
   if (LoadInst *LI = dyn_cast<LoadInst>(P))
     return LI->setOrdering(O);
+  else if (FenceInst *FI = dyn_cast<FenceInst>(P))
+    return FI->setOrdering(O);
```

Comment:
Missing atomicrmw handling?

Reply:
I can add this as well. My understanding is that the bindings tests will not use that, since it uses `LLVMBuildAtomicRMW` to set the ordering directly. Is that okay?